miscellaneous/metaDataRemover/config.toml

```diff
@@ -0,0 +1,5 @@
+ProxyUrl = ""
```

Comment:
might add explanatory comments with some examples for testnet/devnet/mainnet

Reply:
Added explanatory comments